### DISCUSSION TITLE
Screen Window

### DISCUSSION THREAD

**Discussion by snarffff** [videovillage/community#111] [Ideas]
Is it possible to have the UI adaptable, I am using this to check back footage for video blanking (which is usually black) because the interface is dark it makes it pretty hard to see. I am not sure if this is something that has been asked for before, if it is sorry for the repeat. 
To be honest it is getting harder and harder to be able to manually see blanking issues but if there was a way to adapt the interface that may make it a bit easier.

**Comment by gregcotten** [comment]
certainly possible! I was also considering having an optional "edge blanking" mode when zoomed in on an frame, where the black blanking around the image can be another color or checkerboard or something

**Comment by snarffff** [comment]
That sounds great. The program that used towrope best for spotting this on a Mac was quicktime7 pro. That has obviously been dropped because of the 32bit thing. So anything that can adapt so its easier to see edges would be great. 

Andy Kidd
MCR

T +31 20 794 0996
 
 <https://www.glassworksvfx.com/>

 
AMS | BCN | LON
glassworksvfx.com <https://www.glassworksvfx.com/>

GDPR policy details and T&Cs of sale can be found on our website.
Company registered in The Netherlands with number 61618411. Registered office Keizersgracht 477A, 1017DL Amsterdam. VAT NL854415993B01
This email and attachments are strictly privileged, private and confidential and are intended solely for the stated recipient(s). Unauthorised use, copying or disclosure of any of it may be unlawful. If you are not the intended recipient, or if this message has been addressed to you in error, please return to the sender then delete the email and destroy any copies. Please consider the environment before you print.
